src/Kanbas/store/interfaces/quizzes.ts

```diff
@@ -0,0 +1,51 @@
+import { eAssignmentGroup } from "../enums/eAssignmentGroup";
+import { eQuizQuestionType } from "../enums/eQuizQuestionType";
+
+export interface IKanbasQuiz {
+  _id: string;
+  courseId: string; // Course
+  title: string;
+  assignmentGroup: eAssignmentGroup;
+  availableDate: Date;
+  dueDate: Date;
+  points: number;
+  questionsCount: number;
+  isMultipleAvailableDates: boolean;
+  isPublished: boolean;
+}
+
+export interface IKanbasQuizDetails extends IKanbasQuiz {
+  isShuffleAnswers: boolean;
+  timeLimit: number; // minutes
+  isMultipleAttempts: boolean;
+  showCorrectAnswersDate: Date;
+  isOneQuestionAtATime: boolean;
+  isWebcamRequired: boolean;
+  isLockQuestionsAfterAnswering: boolean;
```

Comment:
Do we still need a boolean for isNotifyUsersAboutChange? In the notes, we * to ask professor, did you ask him in his office hours?

Reply:
I forgot I had to ask that. We can keep it for now, and come back to it if needed.